src/style.css

```
@@ -1,3 +1,83 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+:root {
+  --color-blue: #00acdc;
+  --color-black: #000;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.nav__link {
+  text-decoration: none;
+  color: var(--color-black);
+  position: relative;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+}
+
+.nav__link.is-active::after {
```

You can choose a selector like this
```suggestion
.is-active::after {
```